package.json

```diff
@@ -148,5 +151,6 @@
       "types-are-capitalized",
       "types-have-descriptions"
     ]
-  }
+  },
+  "snyk": true
```

Comment:
Do we have a solid convention here? I myself am skeptical of snyk install-time scripting and patching so I usually just try to get the depdendencies themselves updated but don't let snyk mess with this file.

Reply:
We don't have precedent for this.

This is the PR it was introduced with, the language made it seem required to keep the snyk vulnerability testing going, which is why I went with it: https://github.com/reactioncommerce/reaction/pull/6283#issue-488946841